mongox/models.py

```diff
@@ -434,3 +434,15 @@ async def save(self: T) -> T:
         for k, v in self.dict(exclude={"id"}).items():
             setattr(self, k, v)
         return self
+
+    @classmethod
+    async def get_by_id(cls: typing.Type[T], id: str) -> T:
```

Comment:
Can we also accept id as `bson.ObjectId`?
If yes, we can add and extend the tests for it.